flake.nix

```diff
@@ -3,7 +3,7 @@
     flake-utils.url = "github:numtide/flake-utils";
     fedimint = {
       url =
-        "github:fedimint/fedimint?rev=c51ea0c85576008dcbf9d94e2fe994dfa34bca61";
```

Comment:
are you sure? 
rc-3 is https://github.com/fedimint/fedimint/commit/c32bbe8ed8d95ae420b5879e92dda6ce48f5c914

Reply:
the commit you use doesn't have https://github.com/fedimint/fedimint/pull/4726
which was entire point of rc.3

Reply:
Looks like I picked the parent to rc.3 Correcting!